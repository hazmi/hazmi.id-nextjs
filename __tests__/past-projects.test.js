import React from 'react'
import renderer from 'react-test-renderer'
import PastProjects from '../pages/index'

describe('With Snapshot Testing', () => {
  it('show the correct past projects page', () => {
    const component = renderer.create(<PastProjects />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})