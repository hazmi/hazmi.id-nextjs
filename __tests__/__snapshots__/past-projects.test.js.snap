// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing show the correct past projects page 1`] = `
<div
  className="project-list"
>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      Connect33
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="Connect33"
        src="/static/project/connect33.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        A HR mobile application.
      </p>
    </div>
  </div>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      MUTUCare
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="MUTUCare"
        src="/static/project/mutucare.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        A hazard awareness mobile application.
      </p>
    </div>
  </div>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      CelliHealth
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="CelliHealth"
        src="/static/project/cellihealth.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        A healthcare mobile application.
      </p>
    </div>
  </div>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      Vidio
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="Vidio"
        src="/static/project/vidio.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        A social vidio sharing website.
      </p>
    </div>
  </div>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      Liputan6
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="Liputan6"
        src="/static/project/liputan6.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        A news publishing website.
      </p>
    </div>
  </div>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      IN2NITE
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="IN2NITE"
        src="/static/project/in2nite.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        A last minute hotel booking mobile application.
      </p>
    </div>
  </div>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      Kitook
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="Kitook"
        src="/static/project/kitook.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        An Online Travel Agent (OTA) website.
      </p>
    </div>
  </div>
  <div
    className="project-list__item"
  >
    <h3
      className="project-list__title"
    >
      X-Worlds
    </h3>
    <div
      className="project-list__image"
    >
      <img
        alt="X-Worlds"
        src="/static/project/x-worlds.jpg"
        width="100%"
      />
      <span />
    </div>
    <div
      className="project-list__content"
    >
      <p>
        A website that transform kids drawing into reality.
      </p>
    </div>
  </div>
</div>
`;
